apps/examples/wifi_manager/wm_test/wm_test.c

```diff
@@ -843,10 +884,53 @@ int _wt_parse_dns(struct wt_options *opt, int argc, char *argv[])
 	return 0;
 }
 
-int _wt_parse_commands(struct wt_options *opt, int argc, char *argv[])
+int _wt_parse_connectbyrssi(struct wt_options* opt, int argc, char* argv[])
 {
-	int ret = 0;
-	wt_type_e options = _wt_get_opt(argc, argv);
+	if (argc < 6) {
+		return -1;
+	}
+	
+	opt->ssid = argv[2];
+	opt->auth_type = wt_get_auth_type(argv[3]);	
+	if (opt->auth_type == WIFI_MANAGER_AUTH_OPEN || opt->auth_type == WIFI_MANAGER_AUTH_IBSS_OPEN) {
+		// case: open mode
+		opt->password = "";
+		opt->crypto_type = WIFI_MANAGER_CRYPTO_NONE;
+		opt->repeat = atoi(argv[5]);
+		return 0;
+	}
+	
+	// case: security mode + password
+	if (opt->auth_type == WIFI_MANAGER_AUTH_UNKNOWN) {
+		return -1;
+	}
+
+	if (opt->auth_type == WIFI_MANAGER_AUTH_WEP_SHARED) {
+		if ((strlen(argv[5]) == 13) || (strlen(argv[5]) == 26)) {
```

Comment:
Could you please add comment for this length?
For example, `/* XXXX is "CERYTO_WEP_....." */`